exchange/response.go

```diff
@@ -21,6 +21,12 @@ import (
 	"github.com/google/webpackager/resource/preload"
 )
 
+// Keys used in ExtraData.
+const (
+	// See htmltask.ExtractSubContentTypes.
+	SubContentType = "Sub-Content-Type"
```

Comment:
optional: Even though this is only for internal use, "Sub-Content-Type" sounds like a general or standard header and can be confusing (TBH I googled once). Prefix Wpkg- or something?

Reply:
Done